### ISSUE TITLE
Cystine crystalluria

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5527]
missing terms: ,The presence of cystine crystals in the urine.

**Comment by pnrobinson** [comment]
There exists a term for Cystinuria
https://hpo.jax.org/app/browse/term/HP:0003131
Is Cystinuria essentially always associated with crystal formation (in which case we can re-label lthis term) or do we need one term for Cystinuria and one for Cystine crystalluria?

@avizrosenberg
@yongqunh  
@beckysteck

**Comment by pnrobinson** [comment]
On second thought these are measure in different ways and I think having two terms is appropriate. I added a term for Cystine crystalluria.
